roles/iam-user/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Check existance of group policy file.
+  stat:
+    path: "{{playbook_dir}}/opg-ansible/roles/iam-user/files/iam_policy_{{ item }}.json"
```

Comment:
Are sure that the opg-ansible dir will always exist, can we not access the roles path as a variable